Demo/Application/Base/Base View Controllers/BraintreeDemoPaymentButtonBaseViewController.m

```diff
@@ -26,7 +26,7 @@ - (void)viewDidLoad {
         [self.paymentButton.leadingAnchor constraintEqualToAnchor:self.view.leadingAnchor],
         [self.paymentButton.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor],
         [self.paymentButton.centerYAnchor constraintEqualToAnchor:self.view.centerYAnchor constant:self.centerYConstant],
-        [self.paymentButton.heightAnchor constraintEqualToConstant:44.0]
+        [self.paymentButton.heightAnchor constraintEqualToConstant:100.0]
```

Comment:
bumping the height anchor here to account for 3 buttons being displayed on some views, this allows the buttons to be spaced in a way that looks nicer - all other views look the same, this will only imact views with several buttons (NXO, Amex, combined PayPal eventually).
| Before | After |
|-----|-----|
|![Simulator Screenshot - iPhone 14 - 2023-09-08 at 14 09 09](https://github.com/braintree/braintree_ios/assets/20733831/55b92bd8-c044-48d2-9736-60c6af49085b)|![Simulator Screenshot - iPhone 14 - 2023-09-08 at 14 08 55](https://github.com/braintree/braintree_ios/assets/20733831/0de219fd-f015-4af7-9516-4b56866d9e0e)|